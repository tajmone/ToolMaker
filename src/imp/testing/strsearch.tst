%%SET str(klmno)
%%SET str2(Abcklmnoxyz)
%%IF (IMP$STRSEARCH(%%(str),%%(str2),position))
OK!
%%(position)
%%ELSE
FEL!
%%(position)
%%ENDIF

%%IF (IMP$STRSEARCH(%%(str2),%%(str),position))
FEL!
%%(position)
%%ELSE
OK!
%%(position)
%%ENDIF

