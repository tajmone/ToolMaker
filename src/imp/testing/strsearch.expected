########## strsearch ##########
OK!
4

OK!
0

